Kernel/Arch/aarch64/Panic.cpp

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright (c) 2022, Timon Kruiper <timonkruiper@gmail.com>
- *
```

Comment:
i'd squash this into the commit two prior, where the need for this file is removed.

Reply:
Sure thing; done!